diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -407,8 +427,16 @@ namespace diff_drive_controller{
 
   bool DiffDriveController::setOdomParamsFromUrdf(ros::NodeHandle& root_nh,
                              const std::string& left_wheel_name,
-                             const std::string& right_wheel_name)
+                             const std::string& right_wheel_name,
+                             bool lookup_wheel_separation,
+                             bool lookup_wheel_radius)
   {
+    if(not(lookup_wheel_separation or lookup_wheel_radius))
```

Comment:
Even though I think these keywords are awesome, the rest of the project is not using them, so I'd ask you to remove them to keep the consistency of `ros_controllers`